enclone_print/src/print_clonotypes.rs

```diff
@@ -880,54 +851,35 @@ pub fn print_clonotypes(
                     dref,
                     &peer_groups,
                     &mut mlog,
-                    &mut logz,
                     &stats,
                     sr,
                     &extra_args,
                     pcols_sort,
-                    &mut out_data,
+                    &mut res.out_data,
                     &rord,
                     pass,
                     &cdr3_con,
                 );
 
                 // Save.
-
-                res.1.push(logz);
-                res.2.push((exacts.clone(), rsi.clone()));
-                res.12.push(in_center);
+                res.subdata = Some(TraverseResultSubdata {
+                    pic: clonotype_pic,
+                    exacts: exacts.clone(),
+                    rsi: rsi.clone(),
+                    in_center,
+                });
                 for u in 0..exacts.len() {
-                    res.8 += exact_clonotypes[exacts[u]].ncells() as isize;
+                    res.num_cells += exact_clonotypes[exacts[u]].ncells() as isize;
                 }
             }
-            if pass == 2 {
-                res.7.push(out_data);
-            }
         }
+        Ok(res)
     });
-    for r in &results {
-        if !r.13.is_empty() {
-            return Err(r.13.clone());
-        }
-    }
-
-    for ri in &results {
-        for vj in &ri.11 {
-            fate[vj.0].insert(vj.1.clone(), vj.2.clone());
-        }
-    }
```

Comment:
We need this?

Reply:
Oops, good catch!  I tried to fold all of the multiple for-loops into a single pass and it looks like I missed this one.  I will restore this code.

Reply:
Oh, no, I know why this went away - the 11th field of that results tuple was initialized with an empty vector but it was never actually written to anywhere.

Reply:
Put another way - this section of code doesn't actually modify `fate`.  The mutability was only necessary because of this little piece of code which was actually dead, just not in a way that the compiler could notice.  I remove the mutability in a subsequent PR.